### PR TITLE
New Branch against Session/OPM review changes

### PR DESCRIPTION
**Pull request by sarath-sasi-panickaveetil**
> Timezone issue (X axis labelling) in OPM Comparison and OPM charts
> Datalabels in OPM Bar Charts
> Show Y axis in all bar charts(Home, Session and OPM pages
> Datalabels added for 10 mins data in session page.
> Datalabels added for 15 mins data in OPM page bar charts.
> Bar chart bar width increased and reduce the space between bars

### CODE REVIEW

File: src/config/chartConfig.tsx [hunk 1]
```diff
@@ -57,6 +63,10 @@ export const BAR_CHART_OPTIONS: Chart.ChartOptions = {
     },
     tooltip: {
       enabled: false,
+      external: (_) => externalTooltipHandler(_, "session"),
+      mode: "index",
+      external: (_) => externalTooltipHandler(_, "session"),
```
Comment (raj-kumar-litmus) [inline src/config/chartConfig.tsx]: why external and mode are added twice ?

File: src/views/opm.tsx [hunk 2]
```diff
@@ -493,13 +546,14 @@ const OPM: React.FC = () => {
...
+
       {location.pathname.includes("opm") && showFilteredCards && (
         <div
-          className={`flex items-center gap-4 mt-[10px] overflow-scroll ml-[5vw] lg:ml-[0.5vw] ${
-            IS_FULLSCREEN
+          className={`flex items-center gap-4 mt-[10px] overflow-auto ml-[5vw] lg:ml-[1rem] ${IS_FULLSCREEN
               ? "landScape rotate-90 absolute left-[40vw] top-[45vh] ml-[25vw] w-[22vh]"
               : `${width < 700 ? "portrait" : ""}`
-          }`}
+            }`}
```
Comment (raj-kumar-litmus) [inline src/views/opm.tsx:562]: remove white space

File: src/views/opm.tsx [hunk 3]
```diff
@@ -511,7 +565,8 @@ const OPM: React.FC = () => {
                   onClickHandler={removeFormEntry}
                   content={
                     e.type === "time"
-                      ? e.value.toLocaleString("en-US", { hour12: false })
+                      ?
+                      formatDate(e.value, DATE_TIME_FORMAT_2)
```
Comment (raj-kumar-litmus) [inline src/views/opm.tsx]: can this be kept in line #568 ?

File: src/config/chartConfig.tsx (hunk 1 above)
Comment (sarath-sasi-panickaveetil) [reply src/config/chartConfig.tsx]: done

File: src/views/opm.tsx (hunk 3 above)
Comment (sarath-sasi-panickaveetil) [reply src/views/opm.tsx]: done

**Pull request merged**